kickstarts/base.ks.erb

```diff
@@ -103,9 +103,19 @@ EOF
 # Let's rebuild the ramfs with with base scsi drivers we need
 kversion=$(rpm -q kernel --qf '%{version}-%{release}.%{arch}\n')
 ramfsfile="/boot/initramfs-$kversion.img"
-/sbin/dracut --force --add-drivers "mptbase mptscsih mptspi<%= " hv_storvsc hid_hyperv hv_netvsc hv_vmbus" if @target == "hyperv" || @target == "azure" %>" $ramfsfile $kversion
-
-<%= render_partial "post/aws-ec2" if @target == "aws-ec2" %>
+<% case @target
+   when "vsphere", "vagrant" %>
+drivers="mptbase mptscsih mptspi"
+<% when "azure", "hyperv" %>
+drivers="hv_storvsc hid_hyperv hv_netvsc hv_vmbus"
+<% when "ec2" %>
+drivers="xen-blkfront xen-netfront"
+<% else %>
+drivers=""
+<% end %>
+test -n "$drivers" && /sbin/dracut --force --add-drivers "$drivers" $ramfsfile $kversion
```

Comment:
@abellotti we can re-enable it for all platforms if you prefer, but in my view it's slightly cleaner not to. The way I wrote the case statement makes it self documenting and we capture exactly which platforms need which drivers.

Also if those drivers are needed on another platform (pretty sure they aren't), the image won't boot at all. So it we would know very soon in any case.

When I said above vSphere, I meant vSphere and VirtualBox. I checked it and as far as I can see all other platforms use different drivers.